### DISCUSSION TITLE
Get all photos from message

### DISCUSSION THREAD

**Discussion by Dreamer0iQ** [eternnoir/pyTelegramBotAPI#2000] [Q&A] [answered]
I saw solutions how to get one photo from message, but didn't find how to get all photos which was sent. 

I tried to iterate through {message.photo} and download all from this array but this were different resolutions of one image.

**Comment by Badiboy** [comment, marked as answer]
> get all photos

ONE message contains ONE photo. Albums are in fact set of separate messages with common media_group_id.
